### PR TITLE
Helps 1076 make labels optional

### PR DESCRIPTION
**Pull request by RusGraf**

### CODE REVIEW

File: ldk/javascript/examples/self-test-loop/src/config/whisper-group.ts [hunk 1]
```diff
@@ -125,6 +125,12 @@ export const whisperTestGroup = (): TestGroup =>
       10000,
       `No action required`,
     ),
+    new LoopTest(
+      'Whisper Aptitude - Components with no labels',
+      whisperTests.testNoLabels,
+      10000,
+      `NDid components render properly`,
```
Comment (cshields-oliveai) [inline ldk/javascript/examples/self-test-loop/src/config/whisper-group.ts:132]: typo

File: ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts [hunk 2]
```diff
@@ -822,6 +822,81 @@ export const testNumberInputs = (): Promise<boolean> =>
     });
   });
 
+export const testNoLabels = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    try {
+      await whisper.create({
+        label: 'Does it rendered correctly?',
```
Comment (cshields-oliveai) [inline ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts:829]: Did it render correctly?

**Review by cshields-oliveai — APPROVED** [review]

File: ldk/javascript/examples/self-test-loop/src/config/whisper-group.ts (hunk 1 above)
Comment (RusGraf) [reply ldk/javascript/examples/self-test-loop/src/config/whisper-group.ts]: Ahh, my bad

File: ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts (hunk 2 above)
Comment (RusGraf) [reply ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts]: Fixed

**Review by epiccoleman — APPROVED** [review]

**Pull request merged**